.github/actions/comment

```diff
@@ -6,22 +6,41 @@ require_relative "../lib/github"
 stale_label_id = "LA_kwDOEfmk4M8AAAABYVCU-g"
 owner = "community"
 repo = "community"
-only_these_categories = ["Copilot", "Projects and Issues", "Accessibility"]
+only_these_categories = ["Copilot"]
+
+body =<<BODY
+🕒 **Discussion Activity Reminder** 🕒
+
+This Discussion has been labeled as dormant by an automated system for having no activity in the last 60 days. Please consider one the following actions:
+
+1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as `out of date` at the bottom of the page.
+
+2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.
+
+3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.
+
+Note: This dormant notification will only apply to Discussions with the `Question` label. To learn more, see our [recent stalebot announcement](https://github.com/orgs/community/discussions/70478).
```

Comment:
change copy from "[recent stalebot announcement]" to "[recent announcement]" to consistently remove the 'stale' language